src/components/signedInCheck.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {Fragment, useContext} from 'react';
+
+import {CodeContext} from './codeContext';
+
+export function SignedInCheck({
```

Comment:
This component feels kind of poorly named and under-documented.

Can we add a doc comment to this component to better describe what it does.

Reply:
sure thing!